### DISCUSSION TITLE
OLM Newsletter

### DISCUSSION THREAD

**Discussion by xlcrr** [OpenLitterMap/openlittermap-web#77] [Ideas]
Currently, we send out an [email](https://github.com/OpenLitterMap/openlittermap-web/tree/master/resources/views/emails) every ~2 months with the best updates.

However, as we are now open source, a lot more is going to be happening.

I quite like what the OpenStreetMap community did with [weeklyOSM](http://weeklyosm.eu/)

Should we integrate more newsletters or blog.openlittermap.com into OLM? Are weekly updates too frequent? Or should we create a new separate site? Who will maintain it and keep track of the progress, and help make it multilingual?

I would love to hear your thoughts about it.

**Comment by IlanLuci** [comment]
Check out this https://hellboundbloggers.com/setup-custom-domain-name-for-medium-com-blogs/64181/#:~:text=%20%20%201%20Step%201%3A%20Register%20an,you%2012%20A%20records%20that%20you...%20More%20. I thought I had seen this on blogs before and did some research. You can use blog.openlittermap.com as a custom medium domain so people can view your medium articles from that domain.
